src/complex/Core/Application.cpp

```diff
@@ -171,6 +211,10 @@ void Application::loadPlugin(const std::filesystem::path& path, bool verbose)
   getFilterList()->addPlugin(pluginLoader);
 
   auto plugin = pluginLoader->getPlugin();
+
+  m_SIMPL_To_Complex.merge(plugin->getSimplToComplexMap());
```

Comment:
`std::map::merge` skips entries that would be duplicates. We may want to report that a duplicate mapping exists.

Reply:
We should report duplicate entries of complex UUIDs on plugin load (which I think we do?) Or maybe that was in SIMPL where I hit that.

Reply:
We do report duplicate filter uuids but not in this mapping. If one plugin attempts to map a SIMPL filter to the complex one when a mapping already exists, the new mapping will be silently ignored.